psi4/src/psi4/libfunctional/superfunctional.cc

```diff
@@ -381,6 +390,9 @@ void SuperFunctional::add_c_functional(std::shared_ptr<Functional> fun) {
     can_edit();
     c_functionals_.push_back(fun);
 }
+void SuperFunctional::set_dens_tolerance(double cut) {
+    dens_tolerance_=  cut;
+}
```

Comment:
The problem here is that all of the other parameters that you can set do not need to tweak the LibXCFunctional objects and can be handled at the super functional layer which mixed LibXC quantities together. For this function you will need to loop through every functional that is owned by the Superfunctional and call their `set_dens_cutoff` function.

Reply:
Thanks! Setting the cutoff value through SuperFunctional class via python works now.